### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by mcolyer**
Several minor fixes/updates:

* Bumped APM version to latest
* Refactored application.coffee for clarity.
* Used `register('coffee-script').register()` instead of a custom function. @kevinsawicki is that cool?
* Removed unused options that were being passed around.
* Removed some misc scripts from `/scripts/utils`

### CODE REVIEW

**Comment by kevinsawicki** [comment]
> Used register('coffee-script').register() instead of a custom function. @kevinsawicki is that cool?

This will make coffee-script required even if no `.coffee` files are compiled which probably adds `50ms` to startup.

**Comment by mcolyer** [comment]
Ahh, that's why it was there. I'll revert and leave a comment.

File: script/utils/clean-merged-branches [hunk 1]
```diff
@@ -1,41 +0,0 @@
-#!/bin/sh
```
Comment (paulcbetts) [inline script/utils/clean-merged-branches]: Any reason to delete these? They're Still Pretty Neato™

Comment (mcolyer) [inline script/utils/clean-merged-branches]: Just that it was extra code. I've reverted it.

**Comment by mcolyer** [comment]
@paulcbetts :cool: to merge?

**Comment by paulcbetts** [comment]
Thanks @mcolyer!

**Pull request merged**